examples/contrib/gp/sv-dkl.py

```diff
@@ -102,14 +123,27 @@ def main(args):
     deep_kernel = gp.kernels.Warping(rbf, iwarping_fn=cnn)
 
     # init inducing points (taken randomly from dataset)
-    Xu = next(iter(train_loader))[0][:args.num_inducing]
-    # use MultiClass likelihood for 10-class classification problem
-    likelihood = gp.likelihoods.MultiClass(num_classes=10)
-    # Because we use Categorical distribution in MultiClass likelihood, we need GP model returns
-    # a list of probabilities of each class. Hence it is required to use latent_shape = 10.
+    batches = []
+    for i, (data, _) in enumerate(train_loader):
+        batches.append(data)
+        if i >= ((args.num_inducing - 1) // args.batch_size):
+            break
+    Xu = torch.cat(batches)[:args.num_inducing].clone()
```

Comment:
This fixes the logic of the old code, which is wrong when batch_size < num_inducing. We need to draw enough batches.